### PR TITLE
Safely handle schemas without subschemas

### PR DESCRIPTION
**Pull request by LtChae**
smartcitiesdata/react_discovery_ui#73

This also includes formatting on the affected file.

### CODE REVIEW

File: src/components/dataset-dictionary/dataset-dictionary.js [hunk 1]
```diff
@@ -1,105 +1,128 @@
...
-        sortable
-        ExpanderComponent={renderExpander}
-        SubComponent={renderSubTable}
-        expanderDefaults={{ width: expanderWidth }}
-      />
-    </div>
-  )
-}
+const SchemaTable = ({ schema, parentFieldName = "", style }) => {
+  if (schema) {
```
Comment (LtChae) [inline src/components/dataset-dictionary/dataset-dictionary.js]: The functional changes start here

File: src/components/dataset-dictionary/dataset-dictionary.js [hunk 2]
```diff
@@ -1,105 +1,128 @@
...
             {viewLink(datasetId)}
           </div>
-        }
+        )}
       </CollapsableBox>
     </dataset-dictionary>
-  )
-}
+  );
+};
```
Comment (mcsearchin) [inline src/components/dataset-dictionary/dataset-dictionary.js]: Wow.  I didn't know I could *hate* auto-format defaults this much.

File: src/components/dataset-dictionary/dataset-dictionary.js [hunk 3]
```diff
@@ -1,105 +1,128 @@
...
+          sortable
+          ExpanderComponent={renderExpander}
+          SubComponent={renderSubTable}
+          expanderDefaults={{ width: expanderWidth }}
+        />
+      </div>
+    );
+  } else {
+    return (
+      <div>Subschema information not found. Contact the data curator.</div>
```
Comment (mcsearchin) [inline src/components/dataset-dictionary/dataset-dictionary.js]: Test for this change?

Comment (LtChae) [reply src/components/dataset-dictionary/dataset-dictionary.js]: Nary a one. I'll look into it

Comment (mcsearchin) [reply src/components/dataset-dictionary/dataset-dictionary.js]: Cool.  This all looks good otherwise.

Comment (LtChae) [reply src/components/dataset-dictionary/dataset-dictionary.js]: Test added

**Pull request merged**